docs/team/jingyenloh.md

```diff
@@ -3,44 +3,46 @@ layout: page
 title: Loh Jing Yen's Project Portfolio Page
 ---
 
-## Project: AddressBook Level 3
+## Project: ResiReg
 
-AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+**ResiReg** is a desktop residential college management application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 20 kLoC.
```

Comment:
`residential college management application used for teaching Software Engineering principles`

Wow. How can a residential college management application be used for teaching Software Engineering principles? Please enlighten.